webpack.config.js

```diff
@@ -5,8 +5,15 @@ var HTMLWebpackPluginConfig = new HTMLWebpackPlugin({
   inject: 'body'
 });
 
+const path = require('path');
+
 module.exports = {
   entry: __dirname + '/src/index.jsx',
+  performance: {
+    maxEntrypointSize: 512000,
+    maxAssetSize: 512000
+  },
```

Comment:
Resolves the following warnings (displayed by `$ yarn run start`):
```console
WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
This can impact web performance.
Assets: 
  bundle.js (251 KiB)

WARNING in entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
Entrypoints:
  main (251 KiB)
      bundle.js
#...
```